### PR TITLE
Add FXIOS-4739 - Add Github action workflow to update Nimbus' initial experiments

### PR DESCRIPTION
**Pull request by jeddai**
This will add a github action on a cron schedule that will fetch the current firefox_ios first run experiments from the experimenter service, and write them to the `initial_experiments.json` file. If there are changes, it will open a PR for them! On PR closure or merge it will automatically delete the branch.

Issue: #11563 

Examples of it working can be found on my fork: https://github.com/jeddai/firefox-ios/pulls

### CODE REVIEW

File: .github/workflows/update-nimbus-experiments.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: "Update Nimbus Experiments"
+
+on:
+  schedule:
+    - cron: '*/30 * * * *'
```
Comment (nbhasin2) [inline .github/workflows/update-nimbus-experiments.yml:5]: Is there a reason you want to run this every 30min? 

Seems way too often for it to be scheduled for 30 min but if there is a good reason then I am all ears @jeddai

File: .github/workflows/update-nimbus-experiments.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+jobs:
+  update-nimbus-experiments:
+    name: "Update Nimbus Experiments"
+    runs-on: ubuntu-20.04
```
Comment (nbhasin2) [inline .github/workflows/update-nimbus-experiments.yml:10]: any particular reason for using specific ubuntu instance, all other workflows are using `ubuntu-latest` 🤔

Comment (jeddai) [reply .github/workflows/update-nimbus-experiments.yml:10]: Nope! It was based off a different repo's workflows, I'll update it to latest!

File: .github/workflows/update-nimbus-experiments.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    steps:
+      - name: "Checkout Main Branch"
+        uses: actions/checkout@v2
+        with:
+          path: firefox-ios
+          ref: main
+      - name: "Update Experiments JSON"
+        id: update-experiments-json
+        uses: jeddai/update-initial-experiments@v1
```
Comment (nbhasin2) [inline .github/workflows/update-nimbus-experiments.yml:19]: nit: This is something that you are planning to support for long term? 

I am asking as this lies in your repo and is not part of Mozilla's group of repos

Comment (jeddai) [reply .github/workflows/update-nimbus-experiments.yml:19]: I would much prefer it be in the Mozilla-mobile org, is there any guidance for how to start that process/discussion?

**Review by nbhasin2 — APPROVED** [review]
Except for some minor comments, this looks good.

File: .github/workflows/update-nimbus-experiments.yml (hunk 3 above)
Comment (nbhasin2) [reply .github/workflows/update-nimbus-experiments.yml:19]: @jeddai I am not sure who to ask but here is our Mozilla Github guide that might be able to guide you

https://wiki.mozilla.org/GitHub

File: .github/workflows/update-nimbus-experiments.yml (hunk 1 above)
Comment (jaredlockhart) [reply .github/workflows/update-nimbus-experiments.yml:5]: It should be a no-op and exit quickly if there aren't any changes, but the point is the turn around time between the experiment launching and then landing in the repo to make it into the next build.  The major risk of this workflow is missing the timing between launching in Experimenter and the build being cut, so if we set this to something less aggressive like once a day we risk someone thinking they're in time to make it into the build and launching in experimenter and then the build process starting without the change.

We're just prototyping this workflow for now, so this is totally flexible in the future if we find we can get away with checking less frequently, but if it's doable to land at this frequency while we're still figuring out the workflow I think that's safer.

File: .github/workflows/update-nimbus-experiments.yml (hunk 3 above)
Comment (jeddai) [reply .github/workflows/update-nimbus-experiments.yml:19]: https://bugzilla.mozilla.org/show_bug.cgi?id=1784126

Comment (jeddai) [reply .github/workflows/update-nimbus-experiments.yml]: It's been moved and the workflow has been updated! https://github.com/mozilla-mobile/update-initial-experiments

File: .github/workflows/update-nimbus-experiments.yml (hunk 2 above)
Comment (jeddai) [reply .github/workflows/update-nimbus-experiments.yml]: I've updated the runner to be `ubuntu-latest`

**Comment by lmarceau** [comment]
Build is green [here](https://app.bitrise.io/app/6c06d3a40422d10f/pipelines/55ddc228-3641-4a14-b933-5816c9f546a1)
Screen Shot 2022-08-11 at 11 07 01 AM

**Pull request merged**